packages/ods-react/src/components/code/src/components/code/Code.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX, forwardRef } from 'react';
+import { BUTTON_SIZE, BUTTON_VARIANT, Button } from '../../../../button/src';
+import { Clipboard, ClipboardTrigger } from '../../../../clipboard/src';
+import { ICON_NAME, Icon } from '../../../../icon/src';
+import style from './code.module.scss';
+
+interface CodeProp extends ComponentPropsWithRef<'div'> {
+  canCopy?: boolean;
+  labelCopy?: string;
+  labelCopySuccess?: string;
+  onCopy?: () => void,
+}
+
+const Code: FC<CodeProp> = forwardRef(({
+  canCopy = false,
+  className,
+  children,
+  labelCopy,
+  labelCopySuccess,
+  onCopy,
+  ...props
+}, ref): JSX.Element => {
+  return (
+    <div className={ classNames(style[ 'code' ], className) }
+      ref={ ref }
+      { ...props }>
+
+      <pre className={ style[ 'code__preformat' ] }>
+        <code className={ style[ 'code__preformat__code' ] }>
+          { children }
+        </code>
+      </pre>
+
+      <Clipboard
```

Comment:
clipboard should be inside the `canCopy && ...` as otherwise it render the gap on the right event if clipboard is empty